specification/hybridkubernetes/resource-manager/readme.go.md

```diff
@@ -23,4 +25,22 @@ Please also specify `--go-sdks-folder=<path to the root directory of your azure-
 ```yaml $(tag) == 'package-2020-01-01-preview' && $(go)
 namespace: hybridkubernetes
 output-folder: $(go-sdk-folder)/services/preview/$(namespace)/mgmt/2020-01-01-preview/$(namespace)
+```
+### Tag: package-2021-04-01-preview and go
+
+These settings apply only when `--tag=package-2021-04-01-preview --go` is specified on the command line.
+Please also specify `--go-sdks-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+```yaml $(tag) == 'package-2021-04-01-preview' && $(go)
+namespace: hybridkubernetes
+output-folder: $(go-sdk-folder)/services/preview/$(namespace)/mgmt/2021-04-01-preview/$(namespace)
+```
+### Tag: package-2021-03-01 and go
+
+These settings apply only when `--tag=package-2021-03-01 --go` is specified on the command line.
+Please also specify `--go-sdks-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+```yaml $(tag) == 'package-2021-03-01' && $(go)
+namespace: hybridkubernetes
+output-folder: $(go-sdk-folder)/services/preview/$(namespace)/mgmt/2021-03-01/$(namespace)
```

Comment:
This package contains only stable swaggers therefore we must remove the preivew here
```suggestion
output-folder: $(go-sdk-folder)/services/$(namespace)/mgmt/2021-03-01/$(namespace)
```

Reply:
updated it